### PR TITLE
feat(storybook): select icon clickable

### PR DESCRIPTION
**Pull request by bar-tay**

### CODE REVIEW

File: packages/ui-library/src/components/select/index.ts [hunk 1]
```diff
@@ -108,7 +115,7 @@ export class BlrSelect extends LitElementCustom {
         this.sizeVariant,
       ]).toLowerCase() as SizesType;
 
-      const modifiedIcon = this.icon ? this.icon : 'blrChevronDown';
+      const modifiedIcon = this.icon;
```
Comment (angsherpa456) [inline packages/ui-library/src/components/select/index.ts:118]: This is used in only one place and also you can directly use the this.icon so this variable can be removed.

File: packages/ui-library/src/components/select/index.ts [hunk 2]
```diff
@@ -119,9 +126,10 @@ export class BlrSelect extends LitElementCustom {
         },
         {
           'aria-hidden': true,
-        }
+        },
       );
     }
+    return nothing;
```
Comment (angsherpa456) [inline packages/ui-library/src/components/select/index.ts:132]: Is this return necessary?

**Comment by thrbnhrtmnn** [comment]
Hey @bar-tay ,

here are my findings from the review:
- The icon is still shown, even if icon = undefined
- Would it be possible to increase the clickable area of the select so a user can click everywhere inside the outlined area and the menu opens?

File: packages/ui-library/src/components/select/index.ts (hunk 1 above)
Comment (bar-tay) [reply packages/ui-library/src/components/select/index.ts:118]: done

File: packages/ui-library/src/components/select/index.ts (hunk 2 above)
Comment (bar-tay) [reply packages/ui-library/src/components/select/index.ts:132]: done

File: packages/ui-library/src/components/select/index.ts [hunk 3]
```diff
@@ -176,37 +184,39 @@ export class BlrSelect extends LitElementCustom {
...
+                ${this.renderIcon()}
+              </div>
             </div>
-            ${this.renderIcon()}
           </div>
           ${(this.hasHint && this.hintMessage) || (this.hasError && this.errorMessage)
             ? BlrFormCaptionGroupRenderFunction(
                 { theme: this.theme, sizeVariant: this.sizeVariant },
-                this.renderCaptionContent()
+                this.renderCaptionContent(),
```
[the remark this answers is not in the document]
Comment (bar-tay) [reply packages/ui-library/src/components/select/index.ts:219]: done

**Comment by thrbnhrtmnn** [comment]
Hey @bar-tay ,
thanks for the update, this looks good now.

As discussed, lets change 2 minor things:
1. change the default option to the first in the array
2. change the storybook setting, so the parent container is the same as the one we use for input field text & number (using the whole width of the viewport)

**Review by ashk1996 — APPROVED** [review]

**Review by thrbnhrtmnn — APPROVED** [review]

**Pull request merged**